### ISSUE TITLE
docker module fails to parse volumes correctly with read-only set

### ISSUE THREAD

**Issue by ToBeReplaced** [ansible/ansible#8112]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible 1.6.6
##### Environment:

Fedora 20
docker-py 0.3.2
Docker version 1.0.0, build 63fe64c/1.0.0
##### Summary:

`volumes` in the docker module does not accept the read-only flag. Volumes specified with more than one `:` will fail to parse properly.
##### Steps To Reproduce:

``` yaml
- name: Start a docker image with a volume as read only
  docker:
    name=test
    image=centos
    state=running
    volumes=/path/to/local/file:/usr/local/testfile:ro
```
##### Expected Results:

`/usr/local/testfile` should be found in the container.
##### Actual Results:

`/usr/local/testfile` is not found in the container.

The issue is in the logic here:
https://github.com/ansible/ansible/blob/devel/library/cloud/docker#L358

**Comment by mpdehaan** [comment]
Seeing you've identified the line number, something you'd like to submit a pull request to fix, perhaps?

**Comment by ToBeReplaced** [comment]
Is there anything I need to know about the versions of docker or docker-py that ansible will support before proceeding?

If not, I will use these versions for testing:
docker-py 0.3.2
Docker version 1.0.0, build 63fe64c/1.0.0

**Comment by ToBeReplaced** [comment]
#7561 Fixes.

**Comment by ToBeReplaced** [comment]
Also see #6945 which tries to solve the same problem.

**Comment by jimi-c** [comment]
# Closing This Ticket

Hi!

We believe the commit merged in via PR #7561 should resolve this problem for you. This will be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
